llvm/test/CodeGen/AMDGPU/sdwa-peephole-cndmask-fail.ll

```diff
@@ -0,0 +1,40 @@
+; RUN: llc -march=amdgcn -mcpu=gfx1030 < %s | FileCheck %s
```

Comment:
I only see the error with -mtriple=amdgcn-amd-amdhsa

Reply:
Correct. I already fixed that locally, too. Still trying to reduce the mir test case ...